draft-ietf-quic-recovery.md

```diff
@@ -100,30 +100,23 @@ This design obviates the need for disambiguating between transmissions and
 retransmissions and eliminates significant complexity from QUIC's interpretation
 of TCP loss detection mechanisms.
 
-Every packet may contain several frames.  We outline the frames that are
-important to the loss detection and congestion control machinery below.
-
-* Retransmittable frames are those that count towards bytes in flight and need
-  acknowledgement.  The most common are STREAM frames, which typically contain
-  application data.
-
-* Retransmittable packets are those that contain at least one retransmittable
-  frame.
-
-* PADDING frames do cause packets containing them to count towards bytes in
-  flight, but do not elicit acknowledgement.
-  
-* In-flight packets are those that contain at least one retransmittable frame
-  or at least one PADDING frame.  All in-flight packets count towards bytes
-  in flight.
-
-* Cryptographic handshake data is sent in CRYPTO frames, and uses the
-  reliability machinery of QUIC underneath.
-
-* ACK and ACK_ECN frames contain acknowledgment information. ACK_ECN frames
-  additionally contain information about ECN codepoints seen by the peer.  (The
-  rest of this document uses ACK frames to refer to both ACK and ACK_ECN
-  frames.)
+QUIC packets can contain multiple frames of different types. The recovery
+mechanisms ensure that data and frames that need reliable delivery are
+acknowledged or declared lost and sent in new packets as necessary. The types
+of frames contained in a packet affect recovery and congestion control logic:
+
+* All packets are acknowledged, though packets that contain only ACK,
+  ACK_ECN, and PADDING frames are not acknowledged immediately.  Packets
+  containing at least one frame besides ACK, ACK_ECN, and PADDING are referred
+  to as retransmittable below.
+
+* Packets that contain CRYPTO frames are critical to the performance of the
+  QUIC handshake and use shorter timers for acknowledgement and retransmission.
```

Comment:
Technically, this isn't (or need not be) true once you've reached 1-RTT.  Are NewSessionTicket messages treated with equally high priority?